Assignment.py.py

```diff
@@ -0,0 +1,1986 @@
+    index = string.find(sub_string,index+1)
+print(index)
+
+
+# In[12]:
+
+
+#49. find out the index nth occurrence of given substring 
+original_string = input("Enter string: ")
+substring = input("Enter substring: ")
+nth_index = original_string[::-1].find(substring)
+print(len(original_string)-nth_index)
+
+
+# In[4]:
+
+
+#50. Take some single digit numbers from the user and findout min, maximum, sum, average
+single_digit_numbers = list(map(int,input("Enter any single digit numbers: ").split()))
+print("Minimum of all numbers ",min(single_digit_numbers))
+print("Maximum of all numbers ",max(single_digit_numbers))
+print("Sum of all numbers ",sum(single_digit_numbers))
+print("Average of all numbers ",sum(single_digit_numbers)/len(single_digit_numbers))
+
+
+# In[14]:
+
+
+'''51. print the number in proper mathematical way. 
+
+Consider that we have 6 digit numbers. 
+
+Number format  WAP> 10 -> 000010 , 100 ->  000100 , 1000 ->  001000 , 2345678  ->  2345678 
+
+If the number has more than 6 digits then print as it is. '''
+
+number = input("Enter number: ")
+if len(number) >6:
+    print(number)
+else:
+    print("%06d" % int(number))
+
+
+# In[17]:
+
+
+#52. names  ="emp1,emp2,emp3,emp4" iterate through the employee names. 
+names = input("Enter names: ").split(",")
+for name in names:
+    print(name)
+
+
+# In[ ]:
+
+
+'''53. Take actual string, source string, destination string. 
+replce first nth occurrences of source string with destination string of actual string. '''
+
+actual_string = input()
+
+source_string = input()
+dest_string = input()
+
+row = actual_string[::-1].replace(source_string[::-1], dest_string[::-1],1)
+print(row[::-1])
+
+
+# In[ ]:
+
+
+'''54. Take a two numbers from the user and do below menu driven operations 
+
+1. addition, 2. multiples, 3.division , 4.sqrt, 5. pow  a**b , 6.subtraction   
+
+After selection do the corresponding operation. 
+
+Note: user may give int, or float numbers. You should check whether it is proper digits or not. 
+I.e the user given string should be in the position to convert to float. Other wise show the “inproper string given” Error.'''
+import math
+num1 = int(input("Enter number 1: "))
+num2 = int(input("Enter number 2: "))
+if (type(num1)== int or float) and (type(num2) == int or float) :
+    op = int(input("Enter option:\n 1.addition\n 2.subtraction\n 3.multiplication\n 4.division\n 5.sqrt\n 6.pow"))
+    if op == 1:
+        a = int(num1) + int(num2)
+        print("Addition of two numbers: ",a)
+    elif op == 2:
+        s = num1 - num2
+        print("Subtraction of two numbers: ",s)
+    elif op==3:
+        m = num1 * num2
+        print("Multiplication of two numbers: ",m)
+    elif op == 4:
+        d = num1 / num2
+        print("Division of two numbers: ",d)
+    elif op==5:
+        sqt = math.sqrt(num1,num2)
+        print("Sqrt of two numbers: ",sqrt)
+    elif op==6:
+        p = num1 ** num2
+        print("power of two numbers: ", p)
+else:
```

Comment:
it is taking integer values only from the user it is not taking float values read the question again